src/dataframerow/dataframerow.jl

```diff
@@ -5,7 +5,7 @@ immutable DataFrameRow{T <: AbstractDataFrame}
 end
 
 function Base.getindex(r::DataFrameRow, idx::AbstractArray)
-    return DataFrameRow(r.df[[idx]], r.row)
+    return DataFrameRow(r.df[collect(idx)], r.row)
```

Comment:
Can't you simply remove the innner `[ ]` here? `getindex(::DataFrame, idx)` should take care of the rest.